### PR TITLE
Admin resource

### PR DESCRIPTION
**Pull request by tchambers3**
#69 
screen shot 2017-04-30 at 11 27 20 pm

screen shot 2017-04-30 at 11 27 38 pm

### CODE REVIEW

File: app/views/resources/new.html.erb [hunk 1]
```diff
@@ -1,3 +1,4 @@
 <%= render partial: "partials/form_structure", locals: {model_name: "resource",
                                                         object: nil,
                                                         action: "create"} %>
+<%= link_to "Return to this competency's resources", resources_path(:competency_id => @competency_id), class: "waves-effect waves-light btn"  %>
```
Comment (ppuri96) [inline app/views/resources/new.html.erb]: Do we want to standardize these buttons to be large?

File: app/views/resources/edit.html.erb [hunk 2]
```diff
@@ -1,3 +1,5 @@
 <%= render partial: "partials/form_structure", locals: {model_name: "resource",
                                                         object: @resource,
                                                         action: "edit"} %>
+
+<%= link_to "Return to this competency's resources ", resources_path(:competency_id => @competency_id), class: "waves-effect waves-light btn"  %>
```
Comment (ppuri96) [inline app/views/resources/edit.html.erb]: Standardize button to be large? That's how I have it in the Questions model. Can be changed.

File: app/assets/javascripts/scripts.js [hunk 3]
```diff
@@ -311,6 +311,22 @@ function iAgree(){
...
 });
+
+/*
+ * ============================================================================
+ * Tabs Code
+ * ============================================================================
+ */
+
+$(document).ready(function(){
+  $('ul.tabs').tabs();
```
Comment (rahuang) [inline app/assets/javascripts/scripts.js]: I think we already have this at the very top of scripts.js

File: app/assets/javascripts/scripts.js [hunk 4]
```diff
@@ -311,6 +311,22 @@ function iAgree(){
...
  });
 
+ /*
+  * ============================================================================
+  * Collection_select Code
+  * ============================================================================
+  */
+
  $(document).ready(function() {
   $('select').material_select();
```
Comment (rahuang) [inline app/assets/javascripts/scripts.js]: can you move this to the top of the file next to the rest of the other materialize functions

File: app/views/resources/edit.html.erb (hunk 2 above)
Comment (rahuang) [reply app/views/resources/edit.html.erb]: Also just a suggestion on the UI side should we include like a back icon for clarity
like a '<' icon left of "Return to this competency's resources "

File: app/views/resources/index.html.erb [hunk 5]
```diff
@@ -1,3 +1,6 @@
 <%= render partial: "partials/index_structure", locals: {model_name: "resource",
                                                          primary: "active_resources",
                                                          secondary: "inactive_resources"} %>
+
+<%= link_to "Add a New Resource", new_resource_path(:competency_id => @competency_id), class: "waves-effect waves-light btn-large"  %>
```
Comment (rahuang) [inline app/views/resources/index.html.erb]: maybe a plus icon?

**Pull request merged**